tests/unit_tests/test_lib.py

```diff
@@ -515,3 +548,38 @@ def test_global_bounding_box(lib_init):
 
     assert tuple(llc) == expected_llc
     assert tuple(urc) == expected_urc
+
+
+def test_trigger_set_n_batches(lib_run, mpi_intracomm):
+    openmc.reset_auto_ids()
+    pincell = openmc.examples.pwr_pin_cell()
```

Comment:
I went ahead and created a separate fixture with a simplified uo2 model with a k-eff trigger. Let me know if this is works. I couldn't run this test in the same directory as the full pincell model tests since it would try to read the tallies.xml defined by `pincell_model()`